1.0/docs/devguide/properties.md

```diff
@@ -123,7 +123,7 @@ for more information.
 
 The value is interpreted as a method name to be invoked when the property value 
 changes. Note that unlike in 0.5, <strong>property change handlers must be registered 
-explicitly.</strong> The <code><var>propertyName</var>-changed</code> method will not be 
+explicitly.</strong> The <code><var>propertyName</var>Changed</code> method will not be 
```

Comment:
That was the original intention, so +1 to the correction. (If I come up with a clearer way to say this, I might revamp this para.)